autogpt/commands/file_operations.py

```diff
@@ -17,30 +18,92 @@
 CFG = Config()
 
 
-def check_duplicate_operation(operation: str, filename: str) -> bool:
-    """Check if the operation has already been performed on the given file
+def text_checksum(text: str) -> str:
+    """Get the hex checksum for the given text."""
+    digest = hashlib.md5()
```

Comment:
MD5 is actually more than enough for this use case since it's just a checksum, not security-related, not used for unique identification, not used at a large enough scale. Really crc32 would probably be sufficient. 

That being said, md5 should indeed raise a warning in your mind, so I don't want to discourage that.

Reply:
sounds good, thanks for confirming. i still have nightmares from us having to refactor our entire 20 year old codebase out of md5 so I notice it everywhere